src/libtorch.cc

```diff
@@ -1170,19 +1277,23 @@ ModelInstanceState::ProcessRequests(
 
 #ifdef TRITON_ENABLE_GPU
   if (cuda_copy) {
-    cudaStreamSynchronize(stream_);
+    cudaStreamSynchronize(GetCudaStreamByInstanceKind());
     cuda_copy = false;
   }
 #endif
 
   std::vector<torch::jit::IValue> output_tensors;
   uint64_t compute_start_ns = 0;
+  uint64_t compute_infer_start = 0;
 
+  std::tuple<uint64_t*, std::mutex*> compute_infer_cb_data(
+      &compute_infer_start, &timestamp_mu);
```

Comment:
I don't think these changes are required.

Reply:
We are using cuda events for the comput input duration, but we would still need the `compute_infer_start` timestamp for the compute infer duration.